tools/docker/docker-compose.stage.yml

```diff
@@ -148,6 +148,12 @@ services:
   redis:
     image: "docker.io/redis:alpine"
 
+  mailcatcher:
```

Comment:
@Alex-Izquierdo do we need an equivalent one for minikube?

Reply:
I wasn't sure, but if you consider it, I can add it too.

Reply:
@Alex-Izquierdo Should we be modifying these https://github.com/ansible/pinakes/blob/6c77e45523ec662f2e13ee87b08a59d9e7d61be4/tools/keycloak_setup/qa.yml#L47 to point to this mailserver

Reply:
@mkanoor not at all. Just use that smtp server and catches all mails.